Views/Shared/_Layout.cshtml

```diff
@@ -59,6 +59,9 @@
     <script src="~/lib/jquery/dist/jquery.min.js"></script>
     <script src="~/lib/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
     <script src="~/js/site.js" asp-append-version="true"></script>
+    <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css" />
+    <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+    <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
```

Comment:
I would not use jQuery in your scripts. Use jQuery only as required dependency for other libraries.

JQuery was useful in the times when there was no DOM selectors implemented by default in browser. Nowday you do not have to install jQuery, to traverse DOM three.

Read this https://flaviocopes.com/jquery/

Reply:
By the way jQuery is much less performant (i heard even 10 times slower) than native browser API (`querySelector`, `querySelectorAll`)

Reply:
I am using the JQuery for autocomplete functionality.
In text-box for 'Customer Name' and 'Product' in a new purchase, the user will be able to type part of the name and it will retrieve a list of names matching with it. 
Can we use DOM in such a case?

Reply:
Hope yes. :)

Reply:
Autocomplete for sure is possible to implement in native browser API but requires attaching event (onKeyPressed or sth) and custom handler implementation. You can do it as exercise, but if you want to go ahead then leave it.

Reply:
I did not find any sample to use as a reference.
I will let this change on hold and merge the PR, and I will adjust accordingly if I found an example in the future